airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/state/MemoryManager.kt

```diff
@@ -19,7 +19,9 @@ import kotlinx.coroutines.sync.withLock
  * TODO: Some degree of logging/monitoring around how accurate we're actually being?
  */
 @Singleton
-class MemoryManager(availableMemoryProvider: AvailableMemoryProvider) {
+class MemoryManager(
+    private val availableMemoryProvider: AvailableMemoryProvider
```

Comment:
This caused a micronaut bug -- the value wasn't getting injected without `val`

Reply:
weird...

Reply:
Yeah live and learn